variables.tf

```diff
@@ -433,12 +433,6 @@ variable "helm_enabled" {
   type        = bool
 }
 
-variable "helm_version" {
```

Comment:
Excellent. As long as version 3.x is installed from the olcne repo, we can install it.